File: domain/src/main/kotlin/pt/jsal/achman/userstats/UserStats.kt
package pt.jsal.achman.userstats

data class UserStats(
    val totalGames: Int,
    val gamesWithAchievements: Int,
    val totalAchievements: Int,
    val unlockedAchievements: Int,
    val lockedAchievements: Int,
    val completionPercentage: Float,
    val perGameStats: List<UserGameStats>,
)

File: frontend/src/types/userstats/UserStats.ts
import {UserGameStats} from "./UserGameStats.ts";

export type UserStats = {
    totalGames: number;
    gamesWithAchievements: number;
    totalAchievements: number;
    unlockedAchievements: number;
    lockedAchievements: number;
    completionPercentage: number;
    perGameStats: UserGameStats[];
};